wikimedia/wikipedia-ios
# Update ios_login_action funnel

**Phabricator:** https://phabricator.wikimedia.org/T335550

### Notes
* This PR sets the category for the ` login_success` ,`createaccount_start` and `createaccount_success`vactions to `login_to_sync_popover` when they originate from a panel. 
* Previously, it was nil, which would default to the `login` category

### Test Steps
1. Trigger the login panel (an easy way is tapping the watch button on the article when logged out)
2. Login
3. Log out, then repeat step 1, but this time, go on to create an account
4. Verify that the above-mentioned actions have the `login_to_sync_popover` category
